packages/lib/src/dimension-mapper/DimensionMapper.tsx

```diff
@@ -1,29 +1,37 @@
 import { type AxisMapping } from '@h5web/shared/nexus-models';
+import { type HTMLAttributes } from 'react';
 
 import AxisMapper from './AxisMapper';
 import styles from './DimensionMapper.module.css';
 import { type DimensionMapping } from './models';
 import SlicingSlider from './SlicingSlider';
 
-interface Props {
-  className: string;
+interface Props extends Omit<HTMLAttributes<HTMLDivElement>, 'onChange'> {
```

Comment:
Allowing to pass extra HTML attributes for flexibility. I had to omit `onChange` as it was conflicting with the `onChange` prop.